apps/src/templates/projects/FeaturedProjectsTable.jsx

```diff
@@ -5,7 +5,12 @@ import {ImageWithStatus} from '../ImageWithStatus';
 import {Table, sort} from 'reactabular';
 import wrappedSortable from '../tables/wrapped_sortable';
 import orderBy from 'lodash/orderBy';
-import {PROJECT_TYPE_MAP, featuredProjectDataPropType} from './projectConstants';
+import {
+  PROJECT_TYPE_MAP,
+  featuredProjectDataPropType,
+  featuredProjectTableTypes
+}
+from './projectConstants';
```

Comment:
the tiniest nit: `from ./projectConstants;` should go on the line above, like you did in the other file.